scene/resources/visual_shader.cpp

```diff
@@ -3605,7 +3605,7 @@ String VisualShaderNodeOutput::get_input_port_name(int p_port) const {
 	while (ports[idx].mode != Shader::MODE_MAX) {
 		if (ports[idx].mode == shader_mode && ports[idx].shader_type == shader_type) {
 			if (count == p_port) {
-				return String(ports[idx].name).capitalize();
+				return String(ports[idx].name);
```

Comment:
It would be simpler to run it through `EditorPropertyNameProcessor` instead of changing all names to be capitalized manually, no?

Reply:
You may also want to change the generated lower-cased names in `VisualShaderNodeCustom::update_ports()`.

Reply:
@timothyqiu Do you mean changing these port names?
https://github.com/godotengine/godot/blob/2f47a0747cd0ab0c823dee831167d168ec844f11/scene/resources/visual_shader.cpp#L427
https://github.com/godotengine/godot/blob/2f47a0747cd0ab0c823dee831167d168ec844f11/scene/resources/visual_shader.cpp#L445

So instead of "in1" or "out1" it would be "In 1" or "Out 1".

Reply:
Yes. Since `capitalize()` is removed, these should be changed to produce the display name.

Reply:
I only removed `capitalize()` from VisualShaderNodeOutput, not VisualShaderNodeCustom. They don't seem to be related (unless I'm missing something). But I made the change anyways.

Reply:
Ah, sorry, I thought you've touched all the node types :(